data/strings/strings.txt

```diff
@@ -30048,6 +30728,7 @@
     ja = ハイキング
     ko = 하이킹
     lt = Žygiai pėsčiomis
+    lv = Pastaigu maršruti
```

Comment:
This is not only "walking routes". Maybe "Aktīvā atpūta" or simply "Daba"?

Reply:
"Daba" could be confusing (is this trees, rivers etc?), "aktīvā atpūta" seems reasonable.

Reply:
Maybe "Brīvdabas" / "Brīvdabā"?

Reply:
I think we can use "Aktīvā atpūta" to encompass everything.